components/brave_wallet_ui/components/buy-send-swap/select-account-with-header/index.tsx

```diff
@@ -15,8 +15,8 @@ import {
 
 export interface Props {
   accounts: WalletAccountType[]
-  selectedAccount?: UserAccountType
-  onSelectAccount: (account: UserAccountType) => () => void
+  selectedAccount?: WalletAccountType
+  onSelectAccount: (account: WalletAccountType) => void
```

Comment:
I'm not sure why this was a function factory

Reply:
As long as it compiles, it's fine.